charts/asserts/templates/server/configmap.yaml

```diff
@@ -376,5 +385,11 @@ data:
         config:
           root: file:///opt/asserts/api-server/conf/
 
-
+    aws_exporter:
```

Comment:
This is unrelated and needed for the embedded `aws-exporter` code to work in `asserts-server`.